src/mscorlib/shared/System/ValueTuple.cs

```diff
@@ -16,7 +16,8 @@ namespace System
     /// </summary>
```

Comment:
Does F# depend on the behavior of this, too? (see https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/Tuple.cs#L12)

Reply:
It does not.